code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -2262,10 +2262,42 @@ All effects don't start immediately, but rather get worse over time; the rate is
 
 	..()
 
+
+
 /datum/reagent/consumable/ethanol/plasmaflood/on_mob_end_metabolize(mob/living/L)
 	to_chat(L, "<span class='warning'>You no longer feel immune to burning!</span>")
 	. = ..()
 
+/datum/reagent/consumable/ethanol/beeburst
```

Comment:
This needs to go into the monkestation modular folder.
Don't put new changes in the middle of the file